src/components/Registration.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import headerLogo from '../logos/mainLogo.svg';
+import bodyLogo from '../logos/Buildings.svg';
+import './registration.scss'
+
+const Registration = () => {
+  const [user, setUser] = useState({
+    login: '',
+    password: '',
+    passwordRepeat: ''
+  });
+  const verify = (user) => {
```

Comment:
make destructuring here so that there are no such long calls in the function: "user.login.length", but "login.length"